geonode_mapstore_client/client/js/components/SelectInfiniteScroll/SelectInfiniteScroll.jsx

```diff
@@ -106,22 +107,24 @@ function SelectInfiniteScroll({
         }
     }, [page]);
 
+    const SelectComponent = props.creatable ? ReactSelectCreatable : ReactSelect;
+
     return (
-        <SelectSync
+        <SelectComponent
             {...props}
             isLoading={loading}
             options={options}
             onOpen={() => setOpen(true)}
             onClose={() => setOpen(false)}
-            filterOptions={(currentOptions) => {
-                return currentOptions;
-            }}
-            onInputChange={(q) => handleInputChange(q)}
```

Comment:
Why are we removing this when it's creatable?
I think we are loosing the option to filter content